src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -724,7 +725,6 @@ static bool updateAttribute
     {
       attrType = caP->type;
     }
-
```

Comment:
I would not remove these empty lines.
They help readability

Reply:
Fixed in abdaa28